runtime/dotnet/core/ComposerBotPathAdapter.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Extensions.Configuration;
+using System.Collections.Generic;
+
+namespace Microsoft.BotFramework.Composer.Core
+{
+    /// <summary>
+    /// Bot path adapter, for development environment, use '../../' as the bot path, for deployment and production environment, use 'ComposerDialogs' as bot path
+    /// </summary>
+    public static class ComposerBotPathAdapter
+    {
+        public static IConfigurationBuilder UseBotPathAdapter(this IConfigurationBuilder builder, bool isDevelopment = true)
```

Comment:
Naming: consider 'UseBotPathTransformer', 'UseBotPathMapper', 'UseBotPathConverter', since adapter refers to an object oriented pattern that could be confused and is a bit different.